README.md

```diff
@@ -72,6 +72,19 @@ On Windows, reference [librdkafka.redist](https://www.nuget.org/packages/librdka
 For other platforms, follow the source building instructions below.
 
 
+## Building librdkafka - Using vcpkg
+
+You can download and install librdkafka using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
+
+    git clone https://github.com/Microsoft/vcpkg.git
```

Comment:
Use a tripple-tick bash block with `$` prompts to be consistent with the other examples.